src/components/modals/SurveyModal.tsx

```diff
@@ -26,6 +26,7 @@ const SURVEY_OPTS = [
   { label: lstrings.survey_opt_in_person_event, selected: false },
   { label: lstrings.survey_opt_personal_referral, selected: false },
   { label: lstrings.survey_opt_article, selected: false },
+  { label: 'Free Talk Live', selected: false }, // Show name intentionally left untranslated
   { label: lstrings.survey_opt_other_specify, selected: false }
```

Comment:
Our fault. Last minute change to also add "Crypto Canal" as an option. No translation needed as well